### DISCUSSION TITLE
Confliction <Input type="">

### DISCUSSION THREAD

**Discussion by BuchananDevOps** [zeit-ui/react#806] [Ideas]
### Semantic Input
> <input type="text" />  <input type="email" /> <input type="date" /> <= Semantic labeling.

### Geist Conflict 
> type?: "error" | "default" | "secondary" | "success" | "warning" | undefined 

It's kind of annoying to be limited on the input options here.

**Comment by unix** [comment]
Refer to the `htmlType` option in the [Input#API](https://geist-ui.dev/en-us/components/input#apis)  documentation
